core/roboconf-core/src/test/java/net/roboconf/core/agents/DataHelpersTest.java

```diff
@@ -36,63 +39,82 @@
 		props = DataHelpers.readUserData( rawProperties );
 		Assert.assertEquals( null, props.getProperty( DataHelpers.APPLICATION_NAME ));
 		Assert.assertEquals( "root", props.getProperty( DataHelpers.SCOPED_INSTANCE_PATH ));
-		Assert.assertEquals( "192.168.1.24", props.getProperty( DataHelpers.MESSAGING_IP ));
-		Assert.assertEquals( "pwd", props.getProperty( DataHelpers.MESSAGING_PASSWORD ));
-		Assert.assertEquals( "user", props.getProperty( DataHelpers.MESSAGING_USERNAME ));
+		Assert.assertEquals( "192.168.1.24", props.getProperty( MESSAGING_IP ));
+		Assert.assertEquals( "pwd", props.getProperty( MESSAGING_PASSWORD ));
+		Assert.assertEquals( "user", props.getProperty( MESSAGING_USERNAME ));
 
-		rawProperties = DataHelpers.writeUserDataAsString( "192.168.1.24", "user", "pwd", "app", null );
+		rawProperties = DataHelpers.writeUserDataAsString( msgCfg("192.168.1.24", "user", "pwd"), "app", null );
 		props = DataHelpers.readUserData( rawProperties );
 		Assert.assertEquals( "app", props.getProperty( DataHelpers.APPLICATION_NAME ));
 		Assert.assertEquals( null, props.getProperty( DataHelpers.SCOPED_INSTANCE_PATH ));
-		Assert.assertEquals( "192.168.1.24", props.getProperty( DataHelpers.MESSAGING_IP ));
-		Assert.assertEquals( "pwd", props.getProperty( DataHelpers.MESSAGING_PASSWORD ));
-		Assert.assertEquals( "user", props.getProperty( DataHelpers.MESSAGING_USERNAME ));
+		Assert.assertEquals( "192.168.1.24", props.getProperty( MESSAGING_IP ));
+		Assert.assertEquals( "pwd", props.getProperty( MESSAGING_PASSWORD ));
+		Assert.assertEquals( "user", props.getProperty( MESSAGING_USERNAME ));
 
-		rawProperties = DataHelpers.writeUserDataAsString( "192.168.1.24:9120", "user", "pwd", "app", null );
+		rawProperties = DataHelpers.writeUserDataAsString( msgCfg("192.168.1.24:9120", "user", "pwd"), "app", null );
 		props = DataHelpers.readUserData( rawProperties );
 		Assert.assertEquals( "app", props.getProperty( DataHelpers.APPLICATION_NAME ));
 		Assert.assertEquals( null, props.getProperty( DataHelpers.SCOPED_INSTANCE_PATH ));
-		Assert.assertEquals( "192.168.1.24:9120", props.getProperty( DataHelpers.MESSAGING_IP ));
-		Assert.assertEquals( "pwd", props.getProperty( DataHelpers.MESSAGING_PASSWORD ));
-		Assert.assertEquals( "user", props.getProperty( DataHelpers.MESSAGING_USERNAME ));
+		Assert.assertEquals( "192.168.1.24:9120", props.getProperty( MESSAGING_IP ));
+		Assert.assertEquals( "pwd", props.getProperty( MESSAGING_PASSWORD ));
+		Assert.assertEquals( "user", props.getProperty( MESSAGING_USERNAME ));
+	}
+
+	/**
+	 * Creates a pseudo messaging configuration for the given IP and credentials.
+	 * @param ip the pseudo IP address.
+	 * @param user the pseudo user.
+	 * @param pass the pseudo password.
+	 * @return the pseudo messaging configuration.
+	 */
+	private static Map<String, String> msgCfg(String ip, String user, String pass) {
```

Comment:
You wrote a similar function in AgentPropertiesTest.java
Please, make a single one and reuse it.